subliminal/providers/subscenter.py

```diff
@@ -191,6 +192,7 @@ def query(self, title, season=None, episode=None):
                         subtitle_key = subtitle_item['key']
                         downloaded = subtitle_item['downloaded']
                         release = subtitle_item['subtitle_version']
+                        h_version = subtitle_item['h_version']
```

Comment:
I'm not ignoring it, just had little time to review this lately and have no notification that you pushed new commits to this PR. Meanwhile I noticed this other PR too: #714. Does that help? Or did you manage to get the code and tests working?